tools/clang/lib/SPIRV/ModuleBuilder.cpp

```diff
@@ -37,10 +40,10 @@ void ModuleBuilder::GenHeader() {
   TheModule.reserve(kHeaderSize);
   TheModule.emplace_back(spv::MagicNumber);
   TheModule.emplace_back(spv::Version);
-  TheModule.emplace_back(~0u); // TODO: register a generator number
+  TheModule.emplace_back((kGeneratorNumber << 16) | kToolVersion);
```

Comment:
I've just added a unit test for ModuleBuilder. So you'll need update the unit test as well.